auth/server/src/main/java/com/seagates3/util/KeyGenUtil.java

```diff
@@ -63,10 +63,15 @@ public static String createUserId() {
      *
      * @return AccessKeyId
      */
-    public static String createUserAccessKeyId() {
+   public
+    static String createUserAccessKeyId(boolean isPermanent) {
         String id = BinaryUtil.base64UUID().substring(0, 22);
         id = id.replace('-', getRandomChar());
-        id = id.replace('_', getRandomChar());
+        if (isPermanent) {
+          id = "AKIC".concat(id);
```

Comment:
AKIC define in some const file, so that it can be used at other places like UT's

Reply:
Handled